plugins/redacted/client.py

```diff
@@ -250,6 +250,15 @@ def perform_upload(self, payload, torrent_file):
             timeout=self.UPLOAD_TIMEOUT,
         )
 
+
+    def get_user_id(self):
+        return self.request_ajax('index')["id"]
```

Comment:
Please single-quote strings

Reply:
We already have get_index, why not use that directly in get_snatched? get_index()['id'] for example.